### DISCUSSION TITLE
finetune problem

### DISCUSSION THREAD

**Discussion by kuiran** [STVIR/pysot#470] [Q&A]
I want to finetune the pretrainted model use my dataset,but even train 1 epoch.The model can't track the target even in fifth frame.I want to get some suggests how to solve this problem.

**Comment by ZhiyuanChen** [comment]
Can you share with us the training config you used as well as the dataset?
Also, the loss of your fine-tuning process is also mandatory.
